components/Form.js

```diff
@@ -57,11 +64,26 @@ export default function Form({ onCalculatePrice, value }) {
           name="cryptovalue"
           value={num1}
           onChange={handleNumChange1}
-          type="number"
+          type="decimal"
+          min="0"
           placeholder="Type here your crypto value "
           required
         />
+        <DropMenu
+          required
+          onChange={(e) =>
+            setSelectedCoinName(e.target.value, setdisableButton(false))
+          }
```

Comment:
This doesn´t look right. You just want to enable the button after the selection?
Then I would write it this way:
```suggestion
          onChange={(e) => {
            setSelectedCoinName(e.target.value);
            setDisableButton(false); // renamed function before
          }}
```

Reply:
ok have change it. Thx